website/static/files/build_orca_reconcile.sql

```diff
@@ -6,150 +6,141 @@ SET search_path TO orca, public;
-    , orca_etag                   text NOT NULL
-    , s3_etag                     text NULL
-    , orca_last_update            timestamp with time zone NOT NULL
-    , s3_last_update              timestamp with time zone NULL
-    , orca_size                   int8 NOT NULL
-    , s3_size                     int8 NULL
-    , discrepancy_type            text NOT NULL
-    , CONSTRAINT PK_reconcile_catalog_mismatch_report_1 PRIMARY KEY(job_id,collection_id,granule_id,orca_archive_location,key_path)
-    , CONSTRAINT reconcile_jobs_catalog_mismatch_fk FOREIGN KEY(job_id) REFERENCES reconcile_jobs(id)
-    );
 
-    COMMENT ON TABLE reconcile_catalog_mismatch_report
-      IS 'Table that identifies objects that exist in the ORCA catalog and either do not exist in the ORCA S3 bucket or have mismatched values from the s3 objects table.';
-    COMMENT ON COLUMN reconcile_catalog_mismatch_report.job_id
-      IS 'Job the mismatch or missing granule was foundin. Refrences the reconcile_job table.';
-    COMMENT ON COLUMN reconcile_catalog_mismatch_report.collection_id
-      IS 'Cumulus Collection ID value from the ORCA catalog.';
-    COMMENT ON COLUMN reconcile_catalog_mismatch_report.granule_id
-      IS 'Cumulus granuleID value from the ORCA catalog.';
-    COMMENT ON COLUMN reconcile_catalog_mismatch_report.orca_archive_location
-      IS 'ORCA S3 bucket the object is stored in.';
-    COMMENT ON COLUMN reconcile_catalog_mismatch_report.key_path
-      IS 'key path and filename of the object in the ORCA catalog.';
-    COMMENT ON COLUMN reconcile_catalog_mismatch_report.cumulus_archive_location
-      IS 'Expected S3 bucket the object is located in Cumulus. From the ORCA catalog.';
-    COMMENT ON COLUMN reconcile_catalog_mismatch_report.filename
-      IS 'Filename of the object from the ORCA catalog.';
-    COMMENT ON COLUMN reconcile_catalog_mismatch_report.orca_etag
-      IS 'etag of the object as reported in the ORCA catalog.';
-    COMMENT ON COLUMN reconcile_catalog_mismatch_report.s3_etag
-      IS 'etag of the object as reported in the S3 bucket.';
-    COMMENT ON COLUMN reconcile_catalog_mismatch_report.orca_last_update
-      IS 'Last update of the object as reported in the ORCA catalog.';
-    COMMENT ON COLUMN reconcile_catalog_mismatch_report.s3_last_update
-      IS 'Last update of the object as reported in the S3 bucket.';
-    COMMENT ON COLUMN reconcile_catalog_mismatch_report.orca_size
-      IS 'Size in bytes of the object as reported in the ORCA catalog.';
-    COMMENT ON COLUMN reconcile_catalog_mismatch_report.s3_size
-      IS 'Size in bytes of the object as reported in the S3 bucket.';
-    COMMENT ON COLUMN reconcile_catalog_mismatch_report.discrepancy_type
-      IS 'Type of discrepancy found during reconciliation.';
+  CREATE TABLE reconcile_zombie_report  ( 
+    job_id          	int8 NOT NULL,
+    collection_id   	text NOT NULL,
+    granule_id      	text NOT NULL,
+    filename        	text NOT NULL,
```

Comment:
Wondering if we should drop filename since key_path already has that information in it. Maybe shorten key_path to key? Thoughts?